manifest.json

```diff
@@ -5,7 +5,7 @@
   "description": {
     "en": "Kanboard is a simple visual task board web application"
   },
-  "version": "1.2.0-0",
+  "version": "1.2.3-0",
```

Comment:
Could you update the version from the last standard (probably `1.2.3~ynh1`) ?

Reply:
Bump :wink: